src/arrayMethodSort.js

```diff
@@ -4,9 +4,28 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const result = (a, b) => (a.toString() > b.toString());
```

Comment:
```suggestion
  const compareAsStrings = (a, b) => (a.toString() > b.toString());
```
result не слишком подходит для названия функции